RInAppMessaging/Classes/Router.swift

```diff
@@ -75,10 +89,57 @@ internal class Router: RouterType {
                     completion(true)
                     return
                 }
-                view.show(accessibilityCompatible: self.accessibilityCompatibleDisplay, onDismiss: {
+                guard let rootView = self.getKeyWindow(),
+                      self.findPresentedIAMView(from: rootView) == nil else {
+                    return
+                }
+
+                var parentView: UIView = rootView
+
+                // For accessibilityCompatible option, campaign view must be inserted to
```

Comment:
The `accessibilityCompatible` stuff might be clearer if it was isolated in a separate function.

Reply:
Done